packages/xchain-polkadot/src/client.ts

```diff
@@ -130,6 +131,15 @@ class Client implements PolkadotClient, XChainClient {
     }
   }
 
+  validateAddress = (address: string): boolean => {
```

Comment:
Could you add following link into a comment, pls?

https://polkadot.js.org/docs/util-crypto/examples/validate-address

It will help us and others to understand why we validate an address like this ^